src/CodeEditor/index.jsx

```diff
@@ -29,6 +29,10 @@ export default class CodeEditor extends Component
          *  Label position
          */
         labelPosition         : PropTypes.oneOf( [ 'top', 'left', 'right' ] ),
+        /**
+         *  Number of lines shown
```

Comment:
We should change this to something like: `Code editor height (CSS length value)`